src/GafferImage/ColorSpace.cpp

```diff
@@ -102,8 +108,20 @@ void ColorSpace::hashTransform( const Gaffer::Context *context, IECore::MurmurHa
 
 	inputSpacePlug()->hash( h );
 	outputSpacePlug()->hash( h );
+
+	std::string name;
```

Comment:
Actually, scratch that thing about keeping the latter. This whole loop can simply be replaced with `contextPlug()->hash( h )` (although you do need to check for a NULL context plug obviously).